### ISSUE TITLE
CONTRIBUTING.rst mentions outdated "nox -s docs" command

### ISSUE THREAD

**Issue by salty-horse** [googleapis/google-api-python-client#2284]
CONTRIBUTING.rst instructs contributors to build the docs with `nox -s docs`, but this session was removed in #1271.

It is unclear if and how the docs should be regenerated.

**Comment by mljoshi** [comment]
Is this referring to the API documentation currently found in docs/epy?

**Comment by salty-horse** [comment]
No, this is about the `CONTRIBUTING.rst` file at the root of the project tree.

https://github.com/googleapis/google-api-python-client/blob/33762a6405456ca7eae3b5edfbb130eacdda7912/CONTRIBUTING.rst?plain=1#L178

**Comment by daniel-sanche** [comment]
Closing due to inactivity. If this is still a problem, please re-open and we will take a look
